nixos/doc/manual/release-notes/rl-1803.xml

```diff
@@ -247,14 +247,6 @@ following incompatible changes:</para>
       overhead of a go package having non-versioned dependencies.
     </para>
   </listitem>
-  <listitem>
-    <para>
-      When using <option>services.xserver.libinput</option> (enabled by default in GNOME),
```

Comment:
This is from https://github.com/NixOS/nixpkgs/pull/31995, it should not be removed.